### ISSUE TITLE
Errors when events contain utf8 characters

### ISSUE THREAD

**Issue by DanCech** [grafana/devtools#36]
If events contain emojis we end up with error messages like:

```
t=2021-07-09T17:28:25+0000 lvl=eror msg="failed to download file" workerId=7 createdAt=2021-07-09T15:00:00+0000 error="Error 1366: Incorrect string value: '\\xF0\\x9F\\x98\\x84 \"...' for column 'data' at row 1"
```

`\xF0\x9F\x98\x84` is the utf8 representation of 😄 , and it appears that we need to update the database schema and connection character sets to properly handle utf8 characters.

**Comment by marefr** [comment]
Yes, I know :smile: I have a branch somewhere that changes the JSON decoding and don't fail on emojis. I will try to push that up and open a PR. Currently we don't parse the contents such as emojis hence we probably don't need to change the database schema now. Maybe later, but would require re-import everything - the solution is built for that in mind though, but takes quite some time :smile:
